addons/l10n_jo_edi/models/account_move.py

```diff
@@ -64,7 +64,7 @@ def _compute_l10n_jo_edi_is_needed(self):
     def _compute_show_reset_to_draft_button(self):
         # EXTENDS 'account'
         super()._compute_show_reset_to_draft_button()
-        self.filtered(lambda move: move.l10n_jo_edi_state == 'sent').show_reset_to_draft_button = False
+        self.filtered(lambda move: move.l10n_jo_edi_state in move._l10n_jo_edi_state_sent_options()).show_reset_to_draft_button = False
```

Comment:
I am still not a big fan of the demo state.   Plus, if we would think about a demo state, that would be in handy that we can reset to draft so we can delete it, but even then you would have a gap problem in the numbering.